bom-runtime/pom.xml

```diff
@@ -58,11 +58,7 @@
         <groupId>org.lz4</groupId>
         <artifactId>lz4-java</artifactId>
         <version>1.6.0</version>
-      </dependency>
-      <dependency>
```

Comment:
Why remove snappy completely?

Reply:
I think this came up in standup on Friday... @tomekl007 can correct me if I get something wrong but my recollection is that lz4 is a pure Java impl while the snappy impl relies heavily on the native libraries, which seemed to lead back down the path of JNI issues.  Several other plugins wound up excluding this for similar reasons.

Reply:
snappy is not supported in quarkus - other extensions are excluding this as well, for example, kafka-extension. Because it is not supported, there is no need to declare snappy dependency at all.